src/main.rs

```diff
@@ -41,18 +40,37 @@ fn main() {
                      {percent_change_7d}, {last_updated}",
                 ),
         )
+        .arg(
+            Arg::with_name("divider")
+                .short("d")
+                .long("divider")
+                .help("Enter the divider")
+                .takes_value(true)
+                .default_value(" | "),
+        )
         .get_matches();
 
-    let mut url = PathBuf::from("http://api.coinmarketcap.com/v1/ticker/");
-    url.push(matches.value_of("crypto").unwrap());
-
     let client = Client::new();
-    let mut response = client.get(url.to_str().unwrap()).send().unwrap();
+    let mut response = client
+        .get("http://api.coinmarketcap.com/v1/ticker/?limit=10")
+        .send()
+        .unwrap();
     let mut contents = String::new();
     response.read_to_string(&mut contents).unwrap();
-
     let stocks: Vec<HashMap<String, String>> = serde_json::from_str(&contents).unwrap();
```

Comment:
This datatype is very weird, and the method that you're having to use to access it in the correct method is also weird as a result. I'd recommend setting the crypto as the index, and having the individual infos part of the value for the crypto key.